poster/styles/styles.scss

```diff
@@ -104,6 +130,7 @@ ul{
 
 
 // type custom classes
+
 .title__chicago--big{
```

Comment:
BEM suggestion: Update this to `.title__chicago` and then the individual bits could be `.title__chicago--chi`, `.title__chicago--ca`, `.title__chicago--go`. As you have it now, `--big` is a modifier class, but there's no base `.title__chicago`